build.gradle

```diff
@@ -39,6 +39,21 @@ subprojects {
 	apply from: "${rootDir}/gradle/xtend-compiler-settings.gradle"
 	apply from: "${rootDir}/gradle/maven-deployment.gradle"
 	apply from: "${rootDir}/gradle/manifest-gen.gradle"
+
+	configurations.all {
+		if (name.contains("Tooling")) {
+			resolutionStrategy { 
+				eachDependency { DependencyResolveDetails details ->
+					if (details.requested.group == 'org.eclipse.platform' && details.requested.name == 'org.eclipse.core.runtime') {
```

Comment:
Can you add the xtext-dev-bom here instead of a resolution strategy?

Reply:
Will give it a try tomorrow

Reply:
something like https://github.com/eclipse/lsp4j/compare/cd_repairBuild_variantB @oehme ?

Reply:
Yep :+1:

Reply:
updated the PR